### DISCUSSION TITLE
H&M Compability with KAT Rewrite

### DISCUSSION THREAD

**Discussion by Nikodga** [Vdauphin/HeartsAndMinds#1493] [Question and answer]
Hey, a member in my unit has been working for a while in a mission containing H&M. Our unit runs KAT alongside ACE 3 for our ops and we desire to utilize KAT for this mission as well. however, we have found out that the chemical system of KAT is overwriting the chemical system of H&M. We have been looking around the code of KAT and scouring the internet to see if there was any patch available but to no avail we have been successful in finding any kind of solution. Is there a way to make the H&M chemical system work alongside / overwrite the one from KAT?
Thanks
Links of KAT steam wrokshop page provided below
https://steamcommunity.com/workshop/filedetails/?id=2020940806

**Comment by Vdauphin** [comment]
Hello,
May be there is a CBA setting to disable KAT chemical?
Cheers

**Comment by Nikodga** [comment]
Thanks for the speedy response!
About CBA settings, sadly there isn't any setting to disable KAT chemical. That's why i was wondering if there was an option to make the H&M chemical system to take priority in the chemical system

**Reply by Vdauphin** [reply]
> That's why i was wondering if there was an option to make the H&M chemical system to take priority in the chemical system

No this kind of thing is not really possible or is just bad practice. The proper way is to have a setting :/
